src/app/main.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './globals.css';
+import GlobalStyles from './GlobalStyles';
```

Comment:
You need to use GlobalStyles. Now you are just importing it.

Reply:
ahh, I added it to the ReactDOM.render